Examples/Sources/ChatbotExample/Views/LLMSelectionView.swift

```diff
@@ -0,0 +1,164 @@
+import SwiftCrossUI
+import OpenAI
+
+// MARK: - Array Extension for Chunking
+
+extension Array {
+    func chunked(into size: Int) -> [[Element]] {
+        return stride(from: 0, to: count, by: size).map {
+            Array(self[$0..<Swift.min($0 + size, count)])
```

Comment:
I believe that you should just be able to use `min` here instead of `Swift.min`.